recipes-core/images/initramfs-ostree-qcom-image.bb

```diff
@@ -0,0 +1,40 @@
+DESCRIPTION = "Qcom OS OSTree initramfs image"
```

Comment:
Do you need a custom image here? If so, please make it generic, there is nothing qcom specific here.